### ISSUE TITLE
Fix 6% of unknown files; Classify 7,249 .NSDstat files

### ISSUE THREAD

**Issue by raprasad** [IQSS/dataverse#3334]
- similar to #3333

---

7,249 of files with _known_ extension `.NSDstat` are marked as _unknown_.  

Fixes:
- [x] ~~**Now**: Mark these as related to the Norwegian Center for Data Research.~~
  - http://www.nesstar.com/support/faq.html 
- [ ] **Later**: If we can read these files and/or get the DDI, then re-crawl, and save the metadata appropriately
  ##  - Depends on #3088

``` json
 {
        "extension": ".NSDstat",
        "count": 7429,
        "total_count": 124357,
        "percent_string": "5.974%"
  },
```

**Comment by mheppler** [comment]
Related to #2202

Also make sure these files have the proper MIME type to display in the cards and facets.

**Comment by pdurbin** [comment]
Closing in favor of #2202
